### ISSUE TITLE
Jicofo: SEVERE:  An error occurred while executing componentsChanged()

### ISSUE THREAD

**Issue by krassle** [jitsi/docker-jitsi-meet#1471]
Hi! I've updated today from stable-8218 to stable-8252 and was greeted by a suspicious new _WARNING_ and _SEVERE error_ messages:

**WARNING:**
```
Jicofo 2023-02-01 00:09:41.647 WARNING: [22] [xmpp_connection=client] XmppProvider.setComponents#76: Discovered components: [Component(type=end_conference, address=endconference.<no value>), Component(type=speakerstats, address=speakerstats.meet.jitsi), Component(type=room_metadata, address=metadata.meet.jitsi), Component(type=conference_duration, address=conferenceduration.meet.jitsi), Component(type=breakout_rooms, address=breakout.meet.jitsi), Component(type=av_moderation, address=avmoderation.meet.jitsi), Component(type=lobbyrooms, address=lobby.meet.jitsi)]
```

**SEVERE:** 
```
Jicofo 2023-02-01 00:09:41.748 SEVERE: [37] [xmpp_connection=client] XmppProvider.fireComponentsChanged$lambda-5$lambda-4#198: An error occurred while executing componentsChanged() on org.jitsi.jicofo.bridge.BridgeMucDetector@14297859
java.lang.AbstractMethodError: Receiver class org.jitsi.jicofo.bridge.BridgeMucDetector does not define or inherit an implementation of the resolved method 'abstract void componentsChanged(java.util.Set)' of interface org.jitsi.jicofo.xmpp.XmppProvider$Listener.
	at org.jitsi.jicofo.xmpp.XmppProvider.fireComponentsChanged$lambda-5$lambda-4(XmppProvider.kt:196)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
```
After reverting back to the previous stable release the _WARNING_ and _SEVERE_ messages disappeared.

**Comment by saghul** [comment]
@bgrozev anything to worry about?

**Comment by bgrozev** [comment]
It's harmless. Fixed in jitsi/jicofo#1040
